### DISCUSSION TITLE
Change Icon Rustdesk

### DISCUSSION THREAD

**Discussion by Vizeo69** [rustdesk/rustdesk#4535] [Q&A]
Hello,

I can't find where to change this icon 

<img src="https://i.ibb.co/gydSkHX/image-2023-06-02-T07-09-45-496-Z.png" alt="image-2023-06-02-T07-09-45-496-Z" border="0">

I went to this page: https://github.com/CW3MH6/rustdesk-customization 
But I can't find this line: 
"There is also a Base64 encoded icon in the source code. Open /rustdesk/libs/hbb_common/src/config.rs and modify the following lines (starting at line 32) " 

Has this changed?

**Comment by nicolas-ecarnot-sitpi** [comment]
https://github.com/Sitpi/rustdesk/blob/master/flutter/assets/logo.svg
